CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### App Center Distribute
 
-* **[Feature]** Support Swift Package Manager (5.3)
+* **[Feature]** Add the Distribute module to Swift Package Manager. Please note that due to additional resources bundle, it requires Swift 5.3 (Xcode 12 and higher).
```

Comment:
Just a suggestion - not sure about the phrase repeating the word 'distribute' `...Distribute module ... distributed...` :)
```suggestion
* **[Feature]** Add the Distribute module to the package distributed through Swift Package Manager. Please note that due to additional resources bundle, it requires Swift 5.3 (Xcode 12 and higher).
```

Reply:
I am sure that this phrase already enough clear, let's keep it as is.